.gitignore

```diff
@@ -24,4 +24,3 @@ prime/
 snap/.snapcraft/
 stage/
 .sia_temp/
-buildcoveragecoverage.txt
```

Comment:
@poornas did you remove this on purpose?

Reply:
@harshavardhana had a review comment "We don't need this file anymore." , so yes - it was removed on purpose.